testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -48,6 +50,25 @@
     Make_Whole, TPR, GRO, fullerene, two_water_gro,
 )
 
+def test_absence_cutil():
+    with patch.dict('sys.modules', {'MDAnalysis.lib._cutil':None}):
+        if sys.version_info[:2] <= (3, 3):
```

Comment:
Doing robust version comparisons in Python is a little tricky. PEP 440 compliance is the best way, but I'm not sure if we want to add another dependency for that just yet (like `packaging`: https://pypi.org/project/packaging/).

Since we already depend on NumPy, using `LooseVersion` from numpy-distutils might be an option. SciPy still uses that in some places. See examples on StackOverflow: https://stackoverflow.com/a/11887885/2942522

Reply:
> Doing robust version comparisons in Python is a little tricky. PEP 440 compliance is the best way, but I'm not sure if we want to add another dependency for that just yet (like `packaging`: https://pypi.org/project/packaging/).
> 
> Since we already depend on NumPy, using `LooseVersion` from numpy-distutils might be an option. SciPy still uses that in some places. See examples on StackOverflow: https://stackoverflow.com/a/11887885/2942522

Can't we use `sys.hexversion`  as described [here](https://stackoverflow.com/questions/1093322/how-do-i-check-what-version-of-python-is-running-my-script/3018075#3018075). I think it's fairly easy to check using this.

Reply:
You could, though I find those hex version numbers hard to read.